analysis/import/MetricGardenerImporter/src/main/kotlin/de/maibornwolff/codecharta/importer/metricgardenerimporter/model/MetricGardenerNodes.kt

```diff
@@ -3,11 +3,8 @@ package de.maibornwolff.codecharta.importer.metricgardenerimporter.model
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties
 import com.fasterxml.jackson.annotation.JsonProperty
 
-@JsonIgnoreProperties("relationships")
-class MetricGardenerNodes(
-    @JsonProperty("nodes") var metricGardenerNodes: MutableList<MetricGardenerNode>
-                         ) {
-
+@JsonIgnoreProperties("relationships", "info")
```

Comment:
Isn’t there a way to ignore everything besides the nodes property? (Whitelist approach) @fritschldwg